recipes/newrelic/elasticsearch/debian.yml

```diff
@@ -0,0 +1,112 @@
+name: Elasticsearch Open Source Integration
+description: New Relic install recipe for default Elasticsearch Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-elasticsearch
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - elastic
+  - elasticsearch
+
+# CLI runs process detection; this is used to filter recipes that are appropriate for matched processes
+processMatch:
+  - elasticsearch
+
+# Matches partial list of the Log forwarding parameters
+logMatch:
+  - name: elasticsearch
+    file: /var/log/elasticsearch/elasticsearch*.log 
+
+validationNrql: "SELECT count(*) from ElasticsearchClusterSample where hostname like '{{.HOSTNAME}}' SINCE 10 minutes ago"
```

Comment:
We should have a FACET with entityGuid in the query, look in the other recipes for an example.